src/tests/mps_tests.jl

```diff
@@ -1,198 +1,85 @@
 using TensorOperations
```

Comment:
Test files should be in `tests` not `src/tests`

Reply:
done